src/drw_entities.h

```diff
@@ -1143,9 +1143,9 @@ class DRW_Dimension : public DRW_Entity {
     double getDir() const { return rot;}                  /*!< rotation angle of the dimension text, code 53 (optional) default 0 */
     void setDir(const double d) { rot = d;}
 
-    DRW_Coord getExtrusion(){return extPoint;}            /*!< extrusion, code 210, 220 & 230 */
+    DRW_Coord getExtrusion() const {return extPoint;}     /*!< extrusion, code 210, 220 & 230 */
     void setExtrusion(const DRW_Coord p) {extPoint =p;}
-    std::string getName(){return name;}                   /*!< Name of the block that contains the entities, code 2 */
+    std::string getName() const {return name;}            /*!< Name of the block that contains the entities, code 2 */
```

Comment:
this could actually be the following: `std::string const& getName() const {return name;}` which would allow avoiding an unneeded copy in many use cases